### ISSUE TITLE
Versão compatível com Magento 2.4+ e PHP 7.4+

### ISSUE THREAD

**Issue by thiagogalvao** [pagseguro/magento2#151]
Olá, vi atividades recentes no repositório. Quando teremos uma versão compatível com o Magento e PHP mais novos ?

Obrigado.

**Comment by lucaspaiva1** [comment]
Opa, amigo. Acredito que já é compatível com Magento 2.4 e PHP 7.4. 
https://github.com/pagseguro/pagseguro-modulo-magento-v2/pull/141

**Comment by thiagogalvao** [comment]
No processo de instalação pelo composer não avançou porque não era compatível com o PHP 7.4.
Tive que fazer um downgrade pro PHP 7.3

composer.json
"pagseguro/magento2": "^1.12"

**Comment by andreustimm** [comment]
@thiagogalvao conseguiu resolver?

**Comment by thiagogalvao** [comment]
> @thiagogalvao conseguiu resolver?

Não, fiz downgrade para o PHP 7.3 e a versão do Magento fiquei na 2.3.6
Esse componente aqui tá abandonado pela PagSeguro, lamentável.

**Comment by elisei** [comment]
@thiagogalvao tudo bem?

A PagSeguro lançou um novo módulo compatível com 2.4.x disponível aqui:
https://github.com/pagseguro/payment-magento
